src/main/java/io/pinecone/clients/PineconeDataPlaneClient.java

```diff
@@ -0,0 +1,241 @@
+package io.pinecone.clients;
```

Comment:
Can we move this under a new package named "commons" and rename this class to PineconeDataPlaneInterface? This will help the end-user to know that this is not a client where client is a class that the end-user will use.

Reply:
Great call, added `.commons` and updated the naming for the interface.